cmake/Subdirectories.cmake

```
@@ -134,7 +134,8 @@ endfunction()
 #! of the engine.json, project.json, o3de_manifest.json and any gem.json files found visiting
 function(get_all_external_subdirectories output_subdirs)
     # Gather user supplied external subdirectories via the Cache Variable
-    get_property(all_external_subdirs GLOBAL PROPERTY O3DE_EXTERNAL_SUBDIRS)
+    get_property(o3de_external_subdirs GLOBAL PROPERTY O3DE_EXTERNAL_SUBDIRS)
+    list(APPEND all_external_subdirs ${o3de_external_subdirs})
```

Actually can you fix this to grab the O3DE_EXTERNAL_SUBDIRS from the CACHE variable using CACHE PROPERTY
```suggestion
    get_property(o3de_external_subdirs CACHE PROPERTY O3DE_EXTERNAL_SUBDIRS)
    list(APPEND all_external_subdirs ${o3de_external_subdirs})
```